client/go/internal/cli/cmd/config.go

```diff
@@ -307,18 +336,30 @@ func athenzPath(filename string) (string, error) {
 	return filepath.Join(userHome, ".athenz", filename), nil
 }
 
-func (c *Config) loadLocalConfigFrom(parent string) error {
-	home := filepath.Join(parent, ".vespa")
-	_, err := os.Stat(home)
-	if err != nil && !os.IsNotExist(err) {
-		return err
+func (c *Config) findLocalConfigDir(dir string) (string, error) {
+	userHome, err := os.UserHomeDir()
```

Comment:
This broke the use of Vespa CLI to run automated system and staging tests. I guess it's simple to fix that, though.

Reply:
@mpolden should the CLI handle no `$HOME` on UNIX, or is that a user error that shouldn't really happen?